python/pyspark/sql/pandas/types.py

```diff
@@ -993,7 +993,7 @@ def convert_struct(value: Any) -> Any:
 
             def convert_timestamp(value: Any) -> Any:
                 if isinstance(value, datetime.datetime) and value.tzinfo is not None:
-                    ts = pd.Timstamp(value)
+                    ts = pd.Timestamp(value)
```

Comment:
I am fine as is because it's pretty obvious but would be good to have a test at `python/pyspark/sql/tests/test_arrow.py`.

Reply:
+1 for the above comment about adding a test coverage.

Reply:
Yeah I considered adding a test for this but really what we need here is a linter in CI to catch such typos